modular_skyrat/modules/customization/modules/clothing/neck/collars.dm

```diff
@@ -52,6 +52,14 @@
 	greyscale_config_worn = /datum/greyscale_config/collar/choker/worn
 	greyscale_colors = "#222222"
 
+/obj/item/clothing/neck/human_petcollar/thinchoker
+	name = "thin choker"
+	desc = "Like the thicker one, but just a bit thinner!"
```

Comment:
I think this is a bit funnier and catchier.
```suggestion
	desc = "Like the normal one, but thinner!"
```

Reply:
Hell yeah!